q2_sample_classifier/utilities.py

```diff
@@ -126,6 +127,8 @@ def _extract_important_features(index, top):
     # is top a 1-d or multi-d array?
     # coef_ is a multidimensional array of shape = [n_class-1, n_features]
     if any(isinstance(i, list) for i in top) or top.ndim > 1:
+        if issparse(top):
+            top = top.todense()
```

Comment:
That is a straight forward fix!

Reply:
indeed! I was spending too much time trying to figure out why (a) it only affected some methods and (b) interacting with the sparse matrix. In the end it was so much easier to just convert to dense (feature importance scores should not be as massive as the full dataset so I'm hoping this doesn't cause memory issues)